api/src/main/java/org/pmiops/workbench/exfiltration/impl/EgressSumologicRemediationService.java

```diff
@@ -72,9 +74,13 @@ public EgressSumologicRemediationService(
   }
 
   @Override
-  protected void sendEgressRemediationEmail(DbUser user, EgressRemediationAction action)
-      throws MessagingException {
-    mailService.sendEgressRemediationEmail(user, action);
+  protected void sendEgressRemediationEmail(
+      DbUser user, EgressRemediationAction action, DbEgressEvent event) throws MessagingException {
+    SumologicEgressEvent originalEvent = egressEventMapper.toSumoLogicEvent(event);
+    Optional<AppType> appTypeOpt =
+        appServiceNameToAppType(Strings.nullToEmpty(originalEvent.getSrcGkeServiceName()));
+    String serviceName = appTypeOpt.isPresent() ? appTypeOpt.get().toString() : "Jupyter";
```

Comment:
Thanks!
The Optional.get() is not safe in general. However here since I already have `.isPresent() ? appTypeOpt.get()` so it should cause error but not a good practice.  Your PR is a better,  I approved it.